test/test_tutorials.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+path_apphub='../apphub/'
+path_tutorial='../tutorial/'
+path_temp=$(dirname $(mktemp -u))
+
+FILES=$(find ${path_tutorial} -type f -name '*.ipynb')
+cnt=0
+
+for filename in $FILES; do
+    fname=$(basename -- "$filename")
+    extension="${fname##*.}"
+    fname="${fname%.*}"
+    jupyter nbconvert --to script ${filename} --output ${path_temp}${fname}
+    if ipython ${path_temp}${fname}'.py'; then
+        ((cnt=cnt+1))
+    else
+        echo 'failed '${fname}
+        exit 0
+    fi
+done
+rm ${path_temp}/*.py
+echo $cnt 'tests passed out of 10 tests'
```

Comment:
can we infer the total number dynamically instead of hard-coding 10?

Reply:
Yes. updated the script